R/el_cuml.R

```diff
@@ -0,0 +1,132 @@
+#' @title Get an Edgelist From the Specified Network
+#'
+#' @description This function outputs an edgelist from the specified network.
+#'              it chooses the right method depending on the type of network and
+#'              the output is standardly formatted.
+#'
+#' @param dat a Master list object of network models
+#' @param at the current timestep
+#' @param network the index of the network to extract the edgelist from
+#'
+#' @return an edgelist in matrix form with the two columns. Each column
+#'          contains the posit_ids (see \code{get_posit_ids})of the nodes in
+#'          each edge
+#'
+#' @export
+get_edgelist <- function(dat, at, network) {
```

Comment:
this is helpful, but do you then integrate it in `discord_edgelist`?

Reply:
I think I should! I will make sure that it works similarly to [this block](https://github.com/statnet/EpiModel/blob/fix576_cumulative_el/R/net.mod.infection.R#L310-L322) in `discord_edgelist` (I think I missed one possibility in my function) and then use it in `discord_edgelist` as well